src/nextflow.ts

```diff
@@ -0,0 +1,979 @@
+const runname: Fig.Generator = {
+  script: "cat .nextflow/history | awk '{ print $4 }'",
+  postProcess: (output) => {
+    if (output == "") {
+      return [];
+    }
+    return output.split("\n").map((runname) => {
+      return { name: runname.replace("*", "").trim(), description: "Run name" };
+    });
+  },
+};
+
+const completionSpec: Fig.Spec = {
+  name: "nextflow",
+  description: "",
+  parserDirectives: {
+    flagsArePosixNoncompliant: true,
+  },
```

Comment:
Bit of a weird suggestion but I see there are some options eg. `-e.key=value` - we can pretend that `.` is the "option arg separator" which means arg suggestions _should_ work correctly.

```suggestion
  parserDirectives: {
    flagsArePosixNoncompliant: true,
    optionArgSeparators: ["=", "."],
  },
```